launchable/commands/record/session.py

```diff
@@ -118,11 +118,21 @@ def session(
 
     if session_name:
         sub_path = "builds/{}/test_session_names/{}".format(build_name, session_name)
-        res = client.request("get", sub_path)
-
-        if res.status_code != 404:
-            raise click.UsageError(
-                'This session name ({}) is already used. Please set another name.'.format(session_name))
+        try:
+            res = client.request("get", sub_path)
+
+            if res.status_code != 404:
+                click.echo(click.style(
+                    "This session name ({}) is already used. Please set another name."
+                    .format(session_name),
+                    fg='red'),
+                    err=True)
+                sys.exit(2)
+        except Exception as e:
+            if os.getenv(REPORT_ERROR_KEY):
+                raise e
+            else:
+                click.echo(e, err=True)
```

Comment:
When an error happens and just an error is printed, the user will be troubled. 

Also, customers will ask us for an error then, it’s difficult to detect the cause with only an error message, I think.

So I think we should tell that why this message is printed.
e.g) `An error occured on Launchable side, verify request failed. However, the process will continue. Error: {}`

What do you think??

Reply:
## Before

```shell
$ /Users/ono-max/.local/share/virtualenvs/cli-o7XEfK99/bin/launchable record session --build test --session-name hoge
unable to post to hogehoge/intake/organizations/launchableinc/workspaces/mothership/builds/test/test_session_names/hoge
unable to post to hogehoge/intake/organizations/launchableinc/workspaces/mothership/builds/test/test_sessions
cannot access local variable 'session_id' where it is not associated with a value
```

## After

```shell
$ /Users/ono-max/.local/share/virtualenvs/cli-o7XEfK99/bin/launchable record session --build test --session-name hoge
An error occurred on Launchable CLI. You can ignore this message since the process will continue. Error: unable to post to hogehoge/intake/organizations/launchableinc/workspaces/mothership/builds/test/test_session_names/hoge
unable to post to hogehoge/intake/organizations/launchableinc/workspaces/mothership/builds/test/test_sessions
cannot access local variable 'session_id' where it is not associated with a value
```